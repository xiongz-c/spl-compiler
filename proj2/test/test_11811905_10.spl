int test_11811806_10()
{
    float f = 1.0;
    char c = 'a';
    int i = f - c;
    while(10){
        while(10){
            return 1;
        }
    }
    if( 10 > 9 ){
        return 1;
    }else {
        return 2.0;
    }
    return c;
}