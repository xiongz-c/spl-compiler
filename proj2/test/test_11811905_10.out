Error type 7 at Line 5: binary operation on non-number variables
Error type 8 at Line 14: incompatiable return type
Error type 8 at Line 16: incompatiable return type
